docs/user_guide/time_plot.md

```diff
@@ -0,0 +1,101 @@
+The RST plotting routines should have been added to your path during installation, but you can find them in `$RSTPATH/bin`. You can view the available input options for each routine using, for example
```

Comment:
I need to do some work on this file since the `time_plot` functionality has changed since I wrote it. I'll try to do that tomorrow or early next week.

Reply:
Done